OpTestConfiguration.py

```diff
@@ -99,6 +99,8 @@
     'host_password': 'abc123',
     'host_ip': None,
     'secvar_payload_url': 'https://github.com/erichte-ibm/op-test/raw/erichte-ibm/os-secure-boot-squashed/test_binaries',
+    'peer_IP': '100.10.10.15',
+    'priv_IP': '100.10.10.30',
```

Comment:
IP params does it require to be added here ?

Reply:
Correct . These parameters added to config file